xds/src/main/java/io/grpc/xds/internal/security/SecurityProtocolNegotiators.java

```diff
@@ -180,6 +180,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
       extends InternalProtocolNegotiators.ProtocolNegotiationHandler {
     private final GrpcHttp2ConnectionHandler grpcHandler;
     private final SslContextProviderSupplier sslContextProviderSupplier;
+    private volatile boolean handlerRemoved;
```

Comment:
This doesn't need to be volatile. Everything is running on `ctx.executor()`.

Reply:
removed